packages/shared/lib/core/profile-manager/actions/unsubscribeFromWalletApiEvents.ts

```diff
@@ -0,0 +1,7 @@
+import { get } from 'svelte/store'
+
+import { profileManager } from '../stores'
+
+export function unsubscribeFromWalletApiEvents(): void {
+    get(profileManager)?.clearListeners([])
```

Comment:
Shouldn't this be:

```suggestion
    get(profileManager)?.clearListeners(['*'])
```

Otherwise you're not clearing any listeners.

Reply:
I haven't looked under the hood to confirm if the listeners are actually destroyed, but I was just going off of this NodeJS [example](https://github.com/iotaledger/wallet.rs/blob/develop/bindings/nodejs/examples/7-events.js#L61) from the `wallet.rs` repo.